blocks/commerce-cart/commerce-cart.js

```diff
@@ -91,6 +91,37 @@ export default async function decorate(block) {
         Footer: (ctx) => {
           const giftOptions = document.createElement('div');
 
+          if (ctx.item?.itemType === 'ConfigurableCartItem') {
+            const editLinkContainer = document.createElement('div');
+            editLinkContainer.className = 'cart-item-edit-container';
+
+            const editButton = document.createElement('button');
+            editButton.className = 'cart-item-edit-link';
+            editButton.textContent = 'Edit';
+
+            editButton.addEventListener('click', () => {
+              const { item } = ctx;
+              const productUrl = rootLink(`/products/${item.url.urlKey}/${item.topLevelSku}`);
+
+              const params = new URLSearchParams();
+
+              if (item.selectedOptionsUIDs
+                && Array.isArray(item.selectedOptionsUIDs)
+                && item.selectedOptionsUIDs.length > 0) {
+                params.append('optionsUIDs', item.selectedOptionsUIDs.join(','));
+              }
```

Comment:
Since the `item.selectedOptionsUIDs` is an Object, not an array, so this block can't capture the values we need. Also, we only need the encoded values, not the keys. 
![Screenshot 2025-05-15 at 6 20 44 PM](https://github.com/user-attachments/assets/d5b81792-746b-4964-96df-4c3f6ae8d3db)

```suggestion
              if (item.selectedOptionsUIDs) {
                const optionsValues = Object.values(item.selectedOptionsUIDs);

                if (optionsValues.length > 0) {

                  const joinedValues = optionsValues.join(',');
                  params.append('optionsUIDs', joinedValues);
                }
```